sources/packages/api/src/route-controllers/BaseController.ts

```diff
@@ -1,3 +1,6 @@
+import { ApiBearerAuth } from "@nestjs/swagger";
+
+@ApiBearerAuth("access-token")
```

Comment:
Is this really necessary at this level? If yes, I am not sure if all controllers are inheriting from BaseController right now.

Reply:
Yes it is necessary. If I remove it I will get 401 (Unauthorized) whenever I invoke an end point. I put it there so that it will not be required in every controller.